includes/compat/class-wc-stripe-subs-compat.php

```diff
@@ -174,28 +174,6 @@ public function process_payment( $order_id, $retry = true, $force_save_source =
 		}
 	}
 
-	/**
-	 * Overloads WC_Stripe_Payment_Gateway::generate_create_intent_request() in order to
-	 * include additional flags, used when setting payment intents up for off-session usage.
-	 *
-	 * @param WC_Order $order           The order that is being paid for.
-	 * @param object   $prepared_source The source that is used for the payment.
-	 * @return array                    The arguments for the request.
-	 */
-	public function generate_create_intent_request( $order, $prepared_source ) {
-		$request = parent::generate_create_intent_request( $order, $prepared_source );
-
-		// Non-subscription orders do not need any additional parameters.
-		if ( ! $this->has_subscription( $order ) ) {
```

Comment:
This specific check seems to have been missed in the new code, but the path does work. Do you know why the card is saved for off_session usage correctly when checking out with a subscription on this branch?

Reply:
> Do you know why the card is saved for off_session usage correctly when checking out with a subscription on this branch?

Because, on this branch, the card is saved for `off_session` usage **always**. So no need to have specific code for Subscriptions :)